### ISSUE TITLE
accessKey and secret are not send to aws

### ISSUE THREAD

**Issue by barisusakli** [wladiston/nodebb-plugin-amazons3#2]
I was trying to get this to work, but didnt see anywhere in the code where the accessKeyId and secret are passed to aws.

I tried to add it with

```
  AWS.config.update({
       accessKeyId: accessKeyId,
       secretAccessKey: secretAccessKey
  });
```

But no dice, always got the same error 

```
20/1 05:18 [22244] - error: [plugins] filter:uploadImage,  Missing credentials in config
20/1 05:18 [22244] - error: [plugins] filter:uploadImage,  Missing credentials in config
```

Another weird thing I noticed is every image upload results in 2 upload attempts.

**Comment by wladiston** [comment]
I fixed this problem of credentials on version 0.2.1 I guess.
I dont have how to test here where Im.
Could you check it for me?
When I get in my home I'll check this problem of 2 uploads.

**Comment by wladiston** [comment]
That problem of 2 uploads could happen when the file was big. I chanced the plugin to answer the url to the client before finish sending the file upload to the S3
